src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferDetail.cshtml

```diff
@@ -188,4 +200,23 @@
         return string.Join(", ", byDay.Split(",")
             .Select(c => Calendar.DayCodeToName[c]));
     }
+
+    private static string ToDisplay(IEnumerable<AttendingType> attendingTypes)
+    {
+        var orderedDescriptions = attendingTypes
+            .Select(at => at.ToDescription())
+            .Order()
+            .ToArray();
+
+        string display = orderedDescriptions.Length switch
+        {
+            0 => string.Empty,
+            1 => orderedDescriptions[0],
+            2 => string.Join(" and ", orderedDescriptions),
+            _ => string.Join(", ", orderedDescriptions.Take(orderedDescriptions.Length - 2))
+                 + $", {orderedDescriptions[^2]} and {orderedDescriptions[^1]}"
+        };
+
+        return display[0] + display[1..].ToLower();

```

Comment:
This won't work if the display is empty:

var display = string.Empty;
var result = display[0] + display[1..].ToLower();
Console.WriteLine(result);

The above will through an out of range exception!

Reply:
Good spot 👍 
Made the display code generic, added it to the shared kernel and added unit tests.